vendor/vendor.json

```diff
@@ -316,6 +316,78 @@
 			"revision": "2402e8e7a02fc811447d11f881aa9746cdc57983",
 			"revisionTime": "2016-12-17T20:04:45Z"
 		},
+		{
```

Comment:
You can update the `minify` and `parse` packages, there is a big performance improvement with the latest release (see https://github.com/tdewolff/minify/commit/9904fe70e1c1cb5217ed45b24bf9728de3ff10c2). Also, the new `parse` package has incorporated `buffer` and `strconv`, so these dependencies are not required anymore.